Steps proposal Cluster Initialization (#4234)
wazuh/wazuh-documentation: source/installation-guide/wazuh-indexer/step-by-step.rst

The final stage of the process for installing a Wazuh indexer consists in running the security admin script. 

> #. Configure ELASTICSEARCH_IP as an environment variable with the IP address of your Elasticsearch initial node.

Reviewer: ```suggestion
#. On the initial node, configure ELASTICSEARCH_IP as an environment variable.
```
